packages/amazonq/src/lsp/client.ts

```diff
@@ -117,7 +127,7 @@ export async function startLanguageServer(
                 getLogger('amazonqLsp').error('Unable to update bearer token: %s', (e as Error).message)
                 clearInterval(authInterval)
             }
-        }, 300000) // every 5 minutes
+        }, 30 * 1000) // every 30 seconds
```

Comment:
will this `setInterval` be removed as line 122 mentions?

Reply:
It depends on timing. If launch date is before  Flare auth migration, we need to keep this. This is a local JSON RPC call, we can make the interval even smaller.